shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -393,7 +393,7 @@ instance
       pure $ DPState ds ps
 
 data RewardUpdate era = RewardUpdate
-  { deltaT :: !Coin,
+  { deltaT :: !DeltaCoin,
```

Comment:
the reason I had left `deltaT` as `Coin` was because the treasury should only ever be increased by a reward update. is there a reason to change this type?

Reply:
on second thought, maybe `DeltaCoin` is better since it is consistent, and the safety from `Coin` comes from the fact that you can't deserialize negatives (which is irrelevant for the reward update).